### PR TITLE
Add CI workflow to check docs

### PR DESCRIPTION
**Pull request by christophfroehlich**
This should add a CI job with the reusable workflow from https://github.com/ros-controls/control.ros.org/pull/125

Because the workflow is not yet merged, I had to use my fork -> I'll change that before the merge.

### CODE REVIEW

File: .github/workflows/ci-check-docs.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Check Docs
+
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  check-docs:
+    name: Check Docs
+    uses: christophfroehlich/control.ros.org/.github/workflows/reusable-sphinx-check-single-version.yml@reusable_wf
```
Comment (christophfroehlich) [inline .github/workflows/ci-check-docs.yml:10]:
```suggestion
    uses: ros-controls/control.ros.org/.github/workflows/reusable-sphinx-check-single-version.yml@master
```

**Review by olivier-stasse — DISMISSED** [review]
LGTM

**Review by olivier-stasse — APPROVED** [review]
Thanks for the change.

File: .github/workflows/ci-check-docs.yml (hunk 1 above)
Comment (christophfroehlich) [reply .github/workflows/ci-check-docs.yml]: I did already yesterday -> now merged into master

**Pull request merged**